aries-site/src/examples/cardPreviews/header.js

```diff
@@ -12,14 +12,15 @@ export const HeaderPreview = () => {
       <Box direction="row">
         <Button icon={<Hpe color="brand" />} tabIndex={-1} />
         <Box gap="xsmall" direction="row">
-          <Text weight="bold" alignSelf="center">
-            {' '}
-            HPE{' '}
+          <Text color="text-strong" weight="bold" alignSelf="center">
```

Comment:
Nit. We are repeating the `alignSelf`. Should the box just get `align="center"`?

Reply:
Good catch. Let me check it out.